life_dashboard/skills/views.py

```
@@ -1,3 +1,4 @@
 from django.shortcuts import render
+from life_dashboard.skills.models import Skill, SkillCategory
```

Ruff (0.11.9)

`life_dashboard.skills.models.Skill` imported but unused

Remove unused import

(F401)